### ISSUE TITLE
horizontal rulers in latex tables

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#160]
I'd like to have an option for inserting an \hline between all rows of a table, e.g. as an option in the tabularcolumns directive.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/160
- Originally reported by: Anonymous
- Originally created at: 2009-05-12T20:51:41.651

**Comment by shimizukawa** [comment]
_From Wolfram Riedel on 2009-07-03 09:15:46+00:00_

**Bump!**

**Comment by shimizukawa** [comment]
_From Wolfram Riedel on 2009-07-06 09:37:04+00:00_

Maybe Sphinx will get booktabs support like docutils has?

**Comment by shimizukawa** [comment]
_From Wolfram Riedel on 2009-10-09 10:56:27+00:00_

Since the docutils latex writer does have this feature (table_style class) it would seem to be sufficient to (partially) update the latex writer in sphinx.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2010-01-03 14:32:52+00:00_

At some point, that is planned, but it is a lot of work :)

**Comment by shimizukawa** [comment]
_From Nicolas Gruel on 2010-03-25 13:37:52+00:00_

I discovered this problem when comparing between html, rst2pdf and latex version. All the version does have the horizontal line to separate each row in a table but the latex writer.
I did a very ugly hack to have the same behavior between the three writers. I attach the patch to be applied at the file sphinx/writers/latex.py 
I don't think that anybody will be interested but just in case.
